numba_dppy/vectorizers.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2021 Intel Corporation
```

Comment:
Let us follow the same naming convention as Numba, *i.e*, `numba_dppy.np.ufunc.decorators`.  Also, few places have missing docstrings including the module level docstring.

Reply:
It was followed using, cuda in Numba: https://github.com/numba/numba/blob/master/numba/cuda/vectorizers.py

Reply:
Lets keep it simple as is.